packages/lib/src/vis/shared/Html.tsx

```diff
@@ -68,14 +69,17 @@ function Html(props: Props) {
 
   const getInnerDivTransform = useCallback(() => {
     const position = followCamera ? getGroupPosition() : undefined;
+    const scale = v1
+      .setFromMatrixScale(camera.projectionMatrix) // scaling from projected to normalized
+      .multiply(sizeVector.set(width / 2, height / 2, 0)); // scaling from normalized to canvas size
```

Comment:
Technically, I would have to add the scale from world to local but this has no effect as our camera scale is always `1`. 

I don't know if it is worth the added complexity ?

Reply:
Not sure I follow here.

Reply:
Yeah no, no need to add confusion.